SITnett/templates/registration/login.html

```diff
@@ -0,0 +1,24 @@
+{% extends "meny.html" %}
+
+{% block innhold %}
+
+  {% if next %}
+    {% if user.is_authenticated %}
+    Kontoen din har ikke tilgang til denne siden! Logg inn med en gyldig konto.<br>
+    {% else %}
+    Du må logge inn for å se denne siden!<br>
```

Comment:
Stemmer for å droppe utropstegn også her.

Reply:
Enig.